modules/ebi-metagenomics/catpack/addnames/main.nf

```diff
@@ -0,0 +1,56 @@
+process CATPACK_ADDNAMES {
```

Comment:
This one has to be installed with nf-core (I'll adjust that as at the moment is slighly broken because the CGC changes I made)